src/sage/schemes/curves/affine_curve.py

```diff
@@ -2288,17 +2374,17 @@ def _function_field(self):
         return self._nonsingular_model[0]
 
     @lazy_attribute
-    def _lift_to_function_field(self):
+    def _map_to_function_field(self):
```

Comment:
Do we need a deprecated alias when renaming a method/attribute, even if it's non-public?

Reply:
No, as it is non-public.

Reply:
Do you think we need?

Reply:
A relevant excerpt from an old discussion: https://groups.google.com/g/sage-devel/c/PqUKaKVtjOk/m/ztBSPUDfsGIJ

```
... I have always viewed the public interface of a Python module to be the
set of classes and functions that are not private, i.e., do not start
with an underscore. In C++/Java this distinction is even clearer
since the word "public" (and "private") explicitly appears.

Quoting straight from the Python docs [1]: " However, there is a
convention that is followed by most Python code: a name prefixed with
an underscore (e.g. _spam) should be treated as a non-public part of
the API (whether it is a function, a method or a data member). It
should be considered an implementation detail and subject to change
without notice." 
```

Reply:
I agree that we don't need a deprecation.